lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -1,13 +1,42 @@
 ;(function (define, undefined) {
     'use strict';
     define([
-        'gettext', 'jquery', 'underscore', 'backbone', 'js/views/fields'
-    ], function (gettext, $, _, Backbone, FieldViews) {
-
-        var AccountSettingsFieldViews = {};
+        'gettext',
+        'jquery',
+        'underscore',
+        'backbone',
+        'js/views/fields',
+        'text!templates/fields/field_text_account.underscore',
+        'text!templates/fields/field_readonly_account.underscore',
+        'text!templates/fields/field_link_account.underscore',
+        'text!templates/fields/field_dropdown_account.underscore',
+        'text!templates/fields/field_social_link_account.underscore'
+    ], function (
+        gettext, $, _, Backbone,
+        FieldViews,
+        field_text_account_template,
+        field_readonly_account_template,
+        field_link_account_template,
+        field_dropdown_account_template,
+        field_social_link_template)
+    {
+
+        var AccountSettingsFieldViews = {
+            ReadonlyFieldView: FieldViews.ReadonlyFieldView.extend({
+                fieldTemplate: field_readonly_account_template
+            }),
+            TextFieldView: FieldViews.TextFieldView.extend({
+                fieldTemplate: field_text_account_template
+            }),
+            DropdownFieldView: FieldViews.DropdownFieldView.extend({
+                fieldTemplate: field_dropdown_account_template
+            })
+        };
 
         AccountSettingsFieldViews.EmailFieldView = FieldViews.TextFieldView.extend({
```

Comment:
This should be added in the same way you have added 3 keys to AccountSettingsFieldViews at the top of the function.